src/events/message.rs

```diff
@@ -58,6 +60,29 @@ pub async fn message(ctx: client::Context, msg: Message) -> Result<()> {
     Ok(())
 }
 
+async fn handle_emoji_logging(ctx: &client::Context, msg: &Message) -> Result<()> {
+    let guild_emojis = ctx.http.get_emojis(msg.guild_id.unwrap().0).await?;
+    let actual_emojis = util::find_emojis(&msg.content)
+        .iter()
+        .filter_map(|iden| {
+            guild_emojis
+                .iter()
+                .find_map(|a| if a.id == iden.id { Some(a) } else { None })
```

Comment:
why not just `.find`?
```suggestion
                .find(|a| a.id == iden.id)
```

Reply:
yeah its a mistake